### PR TITLE
Add collectd-monasca role that deploys custom collectd code

### PR DESCRIPTION
**Pull request by tpolekhin**
Add collectd-monasca role that deploys custom collectd code to send monasca style metrics to Kafka.

### CODE REVIEW

File: roles/collectd-monasca/defaults/main.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+---
+collectd_image: tpolekhin/collectd-monasca
```
Comment (ryane) [inline roles/collectd-monasca/defaults/main.yml]: is this a custom image? if so, is there a github repo and can it be pushed to the ciscocloud organization instead?

File: roles/collectd-monasca/templates/collectd.conf.j2 [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  Interface "/^veth/"
+  IgnoreSelected true
+</Plugin>
+
+<Plugin "syslog">
+  LogLevel "err"
+</Plugin>
+
+<Plugin "write_kafka">
+  Property "metadata.broker.list" "broker-1.service.consul:9092,broker-2.service.consul:9092,broker-3.service.consul:9092"
```
Comment (ryane) [inline roles/collectd-monasca/templates/collectd.conf.j2]: output should probably be configurable

**Comment by ryane** [comment]
I am wondering if the *monasca roles are applicable to open source mantl or if it would be more appropriate for these addons to live somewhere else. ping @stevendborrelli

**Comment by tymofii-polekhin** [comment]
Updated

**Pull request merged**